challenges/functions.js

```diff
@@ -6,28 +6,43 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+function consume(a, b, cb){
+  return cb(a,b);
+}
 
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+function add(a, b, cb){
+  return cb(a+b);
+}
+
+function multiply(a,b,cb){
+  return cb(a*b) ;
+}
```

Comment:
An additional callback parameter (cb) does not need to be passed into these methods unless you will be passing another function into them at a later time.   

function add(a, b) {
  return a + b;
 }

That is all that is necessary since these functions are being passed to consume.